src/EFCore.Relational/Query/QuerySqlGenerator.cs

```
@@ -1251,4 +1251,14 @@ protected override Expression VisitUnion(UnionExpression unionExpression)
 
         return unionExpression;
     }
+
+    /// <inheritdoc />
+    protected override Expression VisitJsonPathExpression(JsonPathExpression jsonPathExpression)
+    {
+        _relationalCommandBuilder.Append("JSON(");
```

Is this common syntax, better to throw than invalid SQL